### ISSUE TITLE
How to test the model on COCO test-dev set 2017

### ISSUE THREAD

**Issue by Ozone-oo** [leoxiaobin/deep-high-resolution-net.pytorch#210]
I want to test model on test-dev set, but the code pose_estimation/valid.py is only for testing on validation set (5000 images):

So, I have a question: How to test the model on COCO test-dev set 2017 to submit the json file to codalab server?
I'm looking forward from you. Thank you for your great work!

**Comment by tangxiangyu0313** [comment]
> I want to test model on test-dev set, but the code pose_estimation/valid.py is only for testing on validation set (5000 images):
> 
> So, I have a question: How to test the model on COCO test-dev set 2017 to submit the json file to codalab server?
> I'm looking forward from you. Thank you for your great work!
![Uploading image.png…]()
